.evergreen/config.yml

```diff
@@ -93,7 +93,15 @@ functions:
           PROJECT_DIRECTORY: ${PROJECT_DIRECTORY}
         args:
           - .evergreen/run-bundling-test.sh
-
+  run benchmarks:
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: src
+        script: |
+          ${PREPARE_SHELL}
```

Comment:
I'm not seeing a PREPARE_SHELL expansion, I think this is copy pasta from the driver. Can we use subprocess.exec like "run bundling" does?